lib/dialects/mssql/index.js

```diff
@@ -207,7 +207,15 @@ Object.assign(Client_MSSQL.prototype, {
   },
 
   wrapIdentifierImpl(value) {
-    return value !== '*' ? `[${value.replace(/\[/g, '[')}]` : '*';
```

Comment:
`.replace(/\[/g, '[')}]` path was removed because it doesn't seem to do anything. Please correct me if that's wrong.